ts/components/ZendeskSupportComponent.tsx

```diff
@@ -67,30 +73,27 @@ const ZendeskChatComponent = () => {
 
   return (
     <>
-      <ButtonDefaultOpacity
-        onPress={() => openSupportTicket()}
-        transparent={true}
-        testID={"zendeskOpenTicketButton"}
-      >
-        <IconFont
-          name="io-question"
-          color={IOColors.red}
-          testID={"zendeskOpenTicketIcon"}
-        />
-      </ButtonDefaultOpacity>
-      <ButtonDefaultOpacity
-        onPress={() => showSupportTickets()}
-        transparent={true}
-        testID={"zendeskShowTicketsButton"}
-      >
-        <IconFont
-          name="io-chat"
-          color={IOColors.red}
-          testID={"zendeskShowTicketsIcon"}
-        />
-      </ButtonDefaultOpacity>
+      <ButtonWithImage
+        icon={""}
```

Comment:
The icon will be added later?
In any case we should use the appropriate component with the current available data
This is the same but using `ButtonDefaultOpacity`

```typescript
<View>
      <ButtonDefaultOpacity
        style={{
          alignSelf: "stretch"
        }}
        onPress={() => navigation.navigate(navigateToZendeskAskPermissions())}
        disabled={false}
        testID={"contactSupportButton"}
      >
        <Label color={"white"}>
          {I18n.t("support.helpCenter.cta.contactSupport")}
        </Label>
      </ButtonDefaultOpacity>

      <ButtonDefaultOpacity
        onPress={() => {
          showSupportTickets();
          workUnitCompleted();
        }}
        style={{
          alignSelf: "stretch"
        }}
        disabled={false}
        bordered={true}
        testID={"showTicketsButton"}
      >
        <Label>{I18n.t("support.helpCenter.cta.seeReports")}</Label>
      </ButtonDefaultOpacity>
    </View>
```

Reply:
Sorry, my fault! I copied-paste the previous code! Refactored in [ab99c4b](https://github.com/pagopa/io-app/pull/3538/commits/ab99c4bc9c7164d5579339a0e99a5d9108df4bb7)